sdks/python/apache_beam/examples/complete/distribopt/distribopt/distribopt.py

```diff
@@ -0,0 +1,350 @@
+#
```

Comment:
Does this file really has to exist within two `distribopt` subdirectories? Since this is a single file (plus test, but not a dependency), it shouldn't necessarily require the whole `setup.py` file and packaged directory, does it?

Reply:
The only purpose of this structure is to make sure scipy and numpy are installed. I believe numpy is installed by default on dataflow, but what about scipy? I could also have it in a directory and bundle it with a requirements.txt